maps/away/away_site/sensor_relay/sensor_relay.dm

```diff
@@ -10,7 +10,7 @@
 	sectors = ALL_POSSIBLE_SECTORS
 	id = "sensor_relay"
 	template_flags = TEMPLATE_FLAG_SPAWN_GUARANTEED
-	sectors_blacklist = list(SECTOR_LEMURIAN_SEA) //because finding a sensor relay in hell would be weird
+	sectors_blacklist = list(SECTOR_LEMURIAN_SEA, SECTOR_UUEOAESA) //because finding a sensor relay in hell would be weird + there is a specific relay variant in uueoa-esa
```

Comment:
![image](https://github.com/user-attachments/assets/a3569afb-2892-45c1-9f1a-cdc604469cda)
make it like this to be more readable